doc/TRIAGING.md

```diff
@@ -0,0 +1,146 @@
+
+One reason why the role is cycled is that in addition to being generally helpful triaging is a natural way to get exposure to parts of the project that one is not directly involved with, which promotes collaboration, improves the quality of the product and reduces the [bus factor](https://en.wikipedia.org/wiki/Bus_factor).
+
+If the person serving the triage role reaches out to you, please provide all the help that they need. Everyone is generally expected to be helpful to teammates, however when it comes to triage specifically, since the role is cycled, you'll soon be at the other side of the table and it's in your best interest to lead by example.
+
+## Triage in general
+
+The process described in this section and every section below is intended for the person serving the triage role (whether voluntarily or by duty). The process is meant to provide guidance and outline best practices -- not prescribe any rigid patterns that must be followed meticulously. Always use your best judgement and if something reasonable doesn't fit the process, consider amending the process.
+
+Remember to be nice and helpful to people you interact with: answer questions, provide additional context, explain why things are the way the are and how and why we're are going (not) to change them with regards to the given issue.
+
+In a lot of cases the participants are going to be disappointed with our product, since a place for reporting issues naturally skews towards negative feedback, and that disappointment may be very visible. Don't take it personally, chances are, there's indeed an annoying technical issue and the reporter has been struggling with it for a while before deciding to report it, which isn't a particularly entertaining thing to do either. Reporting issues can generally be a draining and thankless process and we don't want to hold up to this expectation. We want people reporting issues to feel welcome, even if they eagerly criticize our work.
+
+However if you don't feel comfortable triaging a certain issue (for any reason, including but not limited to someone in the issue thread being more hostile than you can easily tolerate), do feel free to inform the team of that and we'll figure out whether to assign the issue to a different person or to handle it in some other way.
+
+When serving the triage role, your main responsibility is to move a number of issues forward. We do not mandate any specific number of issues to tackle or any specific amount of time to devote to triaging issues, but here are some general guidelines:
+
+- Try to triage every issue that was created during the time you've been serving the triage role.
+- Try resolve at least one old issue per week. Failing that, make sure to at least move forward a few old issues.
+
```

Comment:
Yes, fantastic idea, thank you. Will do.

> Also, is there a way to label a ticket as high priority?

No. We could add it, but I feel like it would be duplication of information. I added "Low priority" purely for filtering and communication purposes, but high-priority stuff just shouldn't be sitting in the issue, it should be tackled ASAP. So I don't mind adding "High priority", I just don't find it as useful.

Reply:
I see. This just goes with my other comment about handling higher priority tickets first. It would be nice for the triage person to be able to easily find those tickets. I agree that in an ideal world, the label shouldn't be used much!